packages/commonwealth/scripts/async-migrate-thread-version-history.ts

```diff
@@ -0,0 +1,117 @@
+//TODO: This should be deleted after thread version histories are fixed
+import { dispose } from '@hicommonwealth/core';
+import { models, ThreadVersionHistoryAttributes } from '@hicommonwealth/model';
+import { QueryTypes } from 'sequelize';
+
+async function run() {
+  const threadCount = (
+    await models.sequelize.query(
+      `SELECT COUNT(*) FROM "Threads" WHERE version_history_updated = false AND address_id is not NULL`,
+      {
+        raw: true,
+        type: QueryTypes.SELECT,
+      },
+    )
+  )[0];
+
+  const count = parseInt(threadCount['count']);
+  let i = 0;
+  while (i < count) {
+    try {
+      const threadVersionHistory: {
+        id: number;
+        addressId: number;
+        versionHistories: { timestamp: string; body: string; author: string }[];
+      }[] = (
+        await models.sequelize.query(
+          `SELECT id, address_id, version_history FROM "Threads" 
```

Comment:
We need to lock these rows with `FOR UPDATE SKIP LOCKED` and include them in the transaction below. If you don't do this then you risk selecting some rows and edits being made before the script inserts the results into the version history table thus leading to dropped version history.

Need to do this in the comment version history migration script as well.